imap-proto/src/parser/rfc3501/mod.rs

```diff
@@ -249,6 +249,13 @@ fn name_attribute(i: &[u8]) -> IResult<&[u8], NameAttribute> {
         value(NameAttribute::NoSelect, tag_no_case(b"\\Noselect")),
         value(NameAttribute::Marked, tag_no_case(b"\\Marked")),
         value(NameAttribute::Unmarked, tag_no_case(b"\\Unmarked")),
+        value(NameAttribute::All, tag_no_case(b"\\All")),
```

Comment:
Here and in the other instances where all the variants are enumerated, would you mind adding a line comment before each of the RFC 3501, this PR and misc blocks?

Reply:
I have added line comments to separate the name attributes into blocks. That should make it a little easier to tell where the name attributes came from. Is that what you wanted?